x/scheduler/types/msgs.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	sdkerrors "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ sdk.Msg = &MsgUpdateParams{}
```

Comment:
The `schedular` module does not have any params. You can remove this file.

Reply:
this file was deleted.